### PR TITLE
[IMP] website_sale: dont show empty image in cart

### PR DESCRIPTION
**Pull request by shrm-odoo**
Prior this commit:
An empty placeholder image was displayed in the cart when no image was defined for a product.

Post this commit:
The empty placeholder image is no longer displayed in the cart when no image is defined for a product.

task-3983679

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/views/templates.xml [hunk 1]
```diff
@@ -1575,9 +1575,15 @@
                          t-att-src="image_data_uri(line.product_id.image_128)"
                          class="o_image_64_max  img rounded"
                          t-att-alt="line.name_short"/>
-                    <div t-else=""
-                         t-field="line.product_id.image_128"
-                         t-options="{'widget': 'image', 'qweb_img_responsive': False, 'class': 'o_image_64_max rounded'}"/>
+                    <t t-else="">
+                        <t t-set="image_class" t-value="'o_image_64_max rounded %s' % (
+                            'invisible' if not line.product_id.image_128 else '')"/>
```
Comment (kcv-odoo) [inline addons/website_sale/views/templates.xml:1580]: You can use t-attf-value instead of %s

**Review by kcv-odoo — APPROVED** [review]
Added a comment to look at other then that LGTM

File: addons/website_sale/views/templates.xml (hunk 1 above)
Comment (shrm-odoo) [reply addons/website_sale/views/templates.xml:1580]: t-set is not working with t-attf-value

**Review by Feyensv — COMMENTED** [review]
Did you try to add an invisible placeholder without loading the image?
I tested and the image is still fetched, it would be nice if we could avoid fetching it too, to reduce network requests.

File: addons/website_sale/views/templates.xml [hunk 2]
```diff
@@ -1820,9 +1820,15 @@
...
-                         t-options="{'widget': 'image', 'qweb_img_responsive': False, 'class': 'o_image_64_max rounded'}"/>
+                    <t t-else="">
+                        <div t-if="line.product_id.image_128"
+                             t-field="line.product_id.image_128"
+                             t-options="{
+                                'widget': 'image',
+                                'qweb_img_responsive': False,
+                                'class': 'o_image_64_max rounded'}"/>
+                        <div t-else="" class="o_image_64_contain invisible"/>
+                    </t>
```
Comment (Feyensv) [inline addons/website_sale/views/templates.xml:1831]:
```suggestion
                    <div t-elif="line.product_id.image_128"
                             t-field="line.product_id.image_128"
                             t-options="{
                                'widget': 'image',
                                'qweb_img_responsive': False,
                                'class': 'o_image_64_max rounded'}"/>
                    <div t-else="" class="o_image_64_contain invisible"/>
```

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**